cmake/template/setup.bat.in

```diff
@@ -25,3 +25,8 @@ set "YARP_DATA_DIRS=%YARP_DATA_DIRS%;%ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX%/shar
 rem Add the python bindings directory to the PYTHON_PATH
 set "PYTHONPATH=%PYTHONPATH%;%ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX%/lib/python2.7/dist-packages"
 @endcmakeif ROBOTOLOGY_USES_PYTHON
+
+@cmakeif ROBOTOLOGY_USES_SIMULINK
+rem Configure the Matlab path for wb-toolbox
+set "MATLABPATH=%MATLABPATH%;%ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX%/mex;%ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX%/share/WBToolbox;%ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX%/share/WBToolbox/images"
+@endcmakeif ROBOTOLOGY_USES_SIMULINK
```

Comment:
I will split it in a different PR and address your comments there :+1:

### Random rant

I am not a big supporter of this duplication of information (scripts and README). In my opinion we should suggest the users to use only the script, so that if there are env-variables related problems, all users are affected and the problem can be detected. Analogously, if users manually set robotology variables in their environment, a change / addition / removal of the variables might break a system and create confusion. This case should not involve user intervention.